archaius2-core/src/main/java/com/netflix/archaius/ConfigProxyFactory.java

```diff
@@ -98,6 +103,12 @@
 public class ConfigProxyFactory {
     private static final Logger LOG = LoggerFactory.getLogger(ConfigProxyFactory.class);
 
+    // Users sometimes leak both factories and proxies, leading to memory problems that get blamed on us ;-)
```

Comment:
IMHO it would be nice to rewrite the message in lieu of the "true" Netflix culture instead of continuing "blame" culture :)